interface/forms/eye_mag/a_issue.php

```diff
@@ -1368,16 +1368,16 @@ function negate_radio(section) {
                     </tr>
                 </table>
             </div>
-            <p class="text-center" style="width:100%;margin-top:0px;">
+            <div class="text-center w-100 mt-0">
                 <input type="hidden" id="issue_js" name="issue_js" value="test">
                 <input type="hidden" id="pid" name="pid" value="<?php echo attr($pid); ?>">
```

Comment:
Should I add  **/** character and change to ?
```html
<input type="hidden" id="issue_js" name="issue_js" value="test" />
<input type="hidden" id="pid" name="pid" value="<?php echo attr($pid); ?>" />
```

Reply:
@stu01509 Yes